backend/src/main/resources/application-prod.yaml

```diff
@@ -3,4 +3,9 @@ graphql.servlet.cors:
     - https://simplereport.cdc.gov
 simple-report:
   data-hub:
+    uploadenabled: ${datahubUploadEnable:false}
```

Comment:
Complement to my comment on the ConfigurationProperties object: If these could be kebab-case, that would make things easier to read.

Also, this specific property injection seems weird: why not just set it to false and then override with `SIMPLE-REPORT_DATA-HUB_UPLOADENABLED` in the environment, instead of having `DATAHUB_UPLOAD_ENABLE` be a separate magic env var?

Or since it's prod, just assume it's enabled... 😁 

(We could also make all of these things hang off of a bean profile instead—not sure it's useful right now.)